scss/modules/content-types.scss

```diff
@@ -354,28 +358,48 @@
                 .num {
                     color: $grayLight;
                     padding-right: 15px;
+                    display: table-cell;
+                    width: 1%; // make cell fit content
```

Comment:
It works with every width that is smaller than the content in the cell but still greater than 0. Do you prefer pixels? How do you think, does using pixels improve the component?